### ISSUE TITLE
How to use the canopen_motor_node?

### ISSUE THREAD

**Issue by nonlinear1** [ros-industrial/ros_canopen#452]
I am a novice to this package. I have a servo driver with canopen protocol and a BLDC motor.
I want to use this canopen_motor_node to control my motor. I am confused with the all kinds of setting including the ros_control. Could you give a detailed simple example or a detailed explanation? 
Thank you very much

**Comment by mzahana** [comment]
Hi @nonlinear1 

I have the same question. I am also trying to interface the ZLAC3080L motor driver to ROS. The driver supports CANopen. However, there is no enough documentation for this `ros_canopen` package. So, I am not sure how to proceed.
Have you figured out a way to use this package?

Thanks.

**Comment by nonlinear1** [comment]
Hi @mzahana , I am trying to solve it. I have found the following link: https://answers.ros.org/question/250174/how-to-control-maxon-motor-by-using-ros_canopen/ 
Maybe it will give you some help!
I think three configure files are needed: 1)urdf of your robot; 2)driver config files, that is this ros_canopen config files; 3) controller config files, that is the ros_control config files.

**Comment by cschindlbeck** [comment]
I agree that documentation should be more verbose, maybe the README can be updated with minimal working examples or more specific instructions
